stok.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const Hapi = require('hapi');
+
+class Stok {
+  constructor(options) {
+    this.options = options;
```

Comment:
Maybe something like:

```
Stok.config({
  web: {
    port: {
      key: "WEB_PORT",
      default: 3000
    },
  db: {
    host: "DB_HOST",
    user: "DB_USER",
    password: "DB_PASSWORD",
    port: {
      key: "DB_PORT",
      default: 3306
    }
  }
});
```

The structure for the config is arbitrary, but every value must map to an env var, optionally with a default value. This call would then return an object with the same structure and the actual values filled in.

We could also handle default values without using an object by delimiting the env var name and the default value with a character that would never be used in the var names, e.g., `"DB_PORT|3306"` or `"DB_PORT:3306"`.